pkg/crc/validation/validation.go

```diff
@@ -30,19 +31,24 @@ func ValidateMemory(value int) error {
 }
 
 // ValidateBundle checks if provided bundle path exist
-func ValidateBundle(bundle string) error {
-	if err := ValidatePath(bundle); err != nil {
-		if constants.BundleEmbedded() {
-			return errors.Newf("Run 'crc setup' to unpack the bundle to disk")
-		} else {
-			return errors.Newf("Please provide the path to a valid bundle using the -b option")
+func ValidateBundle(bundlePath string) error {
+	// check if this bundle version is cached already, in which case we don't
```

Comment:
`validation.ValidateBundle()` is also used by `crc config set bundle`, and I don't think the newly added shortcut makes sense there, so this will need more work (or more testing)

Reply:
I can confirm that this impacts `crc config set bundle` in a bad way:
```
$ ./crc.git config set bundle /fake/path/crc_libvirt_4.3.1.crcbundle
Successfully configured bundle to /fake/path/crc_libvirt_4.3.1.crcbundle

$ ls -al /fake/path/
ls: cannot access '/fake/path/': No such file or directory
```

Reply:
There is no validation check for the bundle?